File: api-mobile/modules/auth/user.model.js
mongoose = require('mongoose')

const schema = new mongoose.Schema({
    fristname : String,
    lastname: String,
    username: String,
    email: String,
    password: String,
    nohp: String,
    address: String,
    isActive: {type: Boolean, default: true},
    statusDelete: {type: Boolean, default: false}
})

module.exports = mongoose.model("User", schema)

File: vanskediri-frontend/lib/models/user_model.dart
class UserModel {
  String id;
  String firstname;
  String lastname;
  String email;
  String username;
  String nohp;
  String password;
  String address;

  UserModel(
      {this.id,
      this.firstname,
      this.lastname,
      this.nohp,
      this.email,
      this.username,
      this.password,
      this.address});

  UserModel.fromJson(Map<String, dynamic> json) {
    id = json['_id'];
    firstname = json['fristname'];
    lastname = json['lastname'];
    nohp = json['nohp'];
    email = json['email'];
    username = json['username'];
    password = json['password'];
    address = json['address'];
  }

  Map<String, dynamic> toJson() {
    return {
      '_id': id,
      'fristname': firstname,
      'lastname': lastname,
      'email': email,
      'username': username,
      'nohp': nohp,
      'password': password,
      'address': address
    };
  }
}
